3.evaluate_model/scripts/nbconverted/F1_scores.py

```diff
@@ -0,0 +1,91 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+# ### Import Libraries
+
+# In[1]:
+
+
+import pandas as pd
+import pathlib
+from joblib import load
+
+import sys
+sys.path.append("../utils")
+from split_utils import get_features_data
+from train_utils import get_dataset
+from evaluate_utils import model_F1_score
+
+
+# ### Load Necessary Data
+
+# In[2]:
+
+
+# load features data from indexes and features dataframe
+data_split_path = pathlib.Path("../1.split_data/indexes/data_split_indexes.tsv")
+data_split_indexes = pd.read_csv(data_split_path, sep="\t", index_col=0)
+features_dataframe_path = pathlib.Path("../0.download_data/data/training_data.csv.gz")
+features_dataframe = get_features_data(features_dataframe_path)
+
+
+# ### Evaluate Each Model on Each Dataset
+
+# In[3]:
+
+
+# directory to load the models from
+models_dir = pathlib.Path("../2.train_model/models/")
+
+# use a list to keep track of scores in tidy long format for each model and dataset combination
+tidy_scores = []
+
+# iterate through each model (final model, shuffled baseline model, etc)
+for model_path in models_dir.iterdir():
+    model = load(model_path)
+    model_name = model_path.name.replace("log_reg_","").replace(".joblib","")
+    
+    # iterate through label datasets (labels correspond to train, test, etc)
+    # with nested for loops, we test each model on each dataset(corresponding to a label)
+    for label in data_split_indexes["label"].unique():
+        print(f"Evaluating {model_name} on dataset {label}")
+        
+        # load dataset (train, test, etc)
+        data = get_dataset(features_dataframe, data_split_indexes, label)
+        # find model F1 scores on dataset
+        score = model_F1_score(model, data)
+        
+        # add score data to compiled dataframe in tidy format
+        # transpose data and reset index to make dataframe resemble tidy long format
+        score = score.T.reset_index()
+        # change columns to their respective names
+        score.columns = ["Phenotypic_Class", "F1_Score"]
+        # add data split column to indicate which dataset scores are from (train, test, etc)
+        score["data_split"] = label
+        # add shuffled column to indicate if the model has been trained with shuffled data (random baseline) or not
+        score["shuffled"] = True if "shuffled" in model_name else False
```

Comment:
Consider simplifying this if-statement to that of only an `in` comparison (dropping the `if` and `else`), which may yield a similar.

Reply:
Done in https://github.com/WayScience/phenotypic_profiling_model/pull/9/commits/ce807b04f97a7ca330cb5223f4180bf8b61e2dfa